packages/core/src/components/Table/Table.styled.tsx

```diff
@@ -27,7 +27,7 @@ export const TableStyled = styled('table')<TableStyledProps>`
     display: block;
     border: ${({ showRowWithCardStyle, theme }) => !showRowWithCardStyle && `1px solid ${theme.table.borderColor}`};
     padding: ${({ showRowWithCardStyle }) => (showRowWithCardStyle ? '0 0 0.8rem 0' : 0)};
-    border-radius: 0.8rem;
+    border-radius: ${({theme}) => theme.table.borderRadius || '0.8rem'};
```

Comment:
As Mukul pointed out in another file, you can add this `0.8rem` as a default value for `theme.table.borderRadius` and change this line to:

```suggestion
    border-radius: ${({theme}) => theme.table.borderRadius};
```

Reply:
updated accordingly. thanks.